src/RouterSelector.web.js

```diff
@@ -97,18 +105,24 @@ let AppRouter = React.lazy(() => {
 })
 
 const RouterSelector = () => {
+  const [allowUpgradePopup, setAllowUpgradePopup] = useState(false)
```

Comment:
@yaroslav-fedyshyn-nordwhale 
why it was restored ? we've agreed this morning to just add delay to both routers